src/client.rs

```diff
@@ -617,6 +618,38 @@ impl StatsdClient {
         Ok(StatsdClient::builder(prefix, sink).build())
     }
 
+    /// Create a new client instance that will use the given prefix to send
+    /// metrics to the given host over UDS using an appropriate sink.
+    ///
+    /// The created UDS socket will be put into non-blocking mode.
+    ///
+    /// Note that this client will discard errors encountered when
+    /// sending metrics via the `MetricBuilder::send()` method.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use cadence::{StatsdClient, UdsMetricSink};
+    ///
+    /// let prefix = "my.stats";
+    /// let path = "/tmp/sock";
+    ///
+    /// let client = StatsdClient::from_uds_path(prefix, path);
+    /// ```
+    ///
+    /// # Failures
+    ///
+    /// This method may fail if:
+    ///
+    /// * It is unable to create a local UDS socket.
+    /// * It is unable to put the UDS socket into non-blocking mode.
+    pub fn from_uds_path(prefix: &str, addr: &str) -> MetricResult<Self> {
```

Comment:
Do you mind removing this method? I now feel like the `from_udp_host` method was a mistake (though I haven't gotten around to marking it as deprecated yet). It's since been superseded by the `from_sink` and builder constructors.